app/src/components/adminConsole/publishedCups/PublishedCupsHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { SectionHeader } from "../AdminConsoleStyles";
+import Elle from "../../../images/Elle";
+import styled from "styled-components";
+
+export const MiniElleImg = styled(Elle)`
+    height: 66px;
+    width: auto;
+    vertical-align: text-bottom;
+    margin-right: 19px;
+`;
+
+const PublishedCupsHeader: React.FunctionComponent = () => {
+    return (
+        <div id="currentCupHeader">
+            <SectionHeader>
+                <MiniElleImg />
+                Published SnapCups
+            </SectionHeader>
```

Comment:
I think these four lines are the basic of a reusable section header component.
Nearly this exact same thing exists in `CurrentCupHeader`. Could you extra a reusable common component?

Reply:
Yup - it's now defined for Your Snaps and Published Cups. It's not really feasible for 'current cup' because of the extra button but MiniElleImg is now there as a component